ci/gpu/build.sh

```diff
@@ -43,7 +43,9 @@ nvidia-smi
 logger "Activate conda env..."
 source activate gdf
 conda install "rmm=$MINOR_VERSION.*" "nvstrings=$MINOR_VERSION.*" "cudatoolkit=$CUDA_REL" \
-              "dask>=2.0" "distributed>=2.0" "numpy>=1.16"
+              "dask>=2.1.0" "distributed>=2.1.0" "numpy>=1.16" "double-conversion" \
+              "rapidjson" "flatbuffers" "boost-cpp" "fsspec>=0.3.3" "dlpack" \
+              "feather-format" "cupy>=6.0.0"
```

Comment:
PR opened regarding this here: https://github.com/rapidsai/gpuci-build-environment